packages/react-next/src/components/Slider/Slider.test.tsx

```diff
@@ -2,13 +2,11 @@ import * as React from 'react';
 import * as ReactDOM from 'react-dom';
 import { create } from '@uifabric/utilities/lib/test';
 import * as ReactTestUtils from 'react-dom/test-utils';
-import * as path from 'path';
```

Comment:
Bad merge? (also the change to sharedIsConformant)

Reply:
No; with the copy of the `/common` folder, `Slider.test.tsx` began failing without those changes. Since it was the only file that used `sharedIsConformant`, I simply removed the file.

Reply:
`sharedIsConformant` is our entry point for the new conformance tests we're currently working on adding. if it's broken it needs to be fixed, not removed. And I'm confused how any of the changes you made in this PR could cause it to start failing for an unrelated component.